src/dgraph.ts

```diff
@@ -1,11 +1,16 @@
 import fetch from 'node-fetch';
 import { GraphQLResponse, AuthHeaderField } from '@slash-graphql/lambda-types';
 
-export async function graphql(query: string, variables: Record<string, any> = {}, authHeader?: AuthHeaderField): Promise<GraphQLResponse> {
+export async function graphql(query: string, variables: Record<string, any> = {}, authHeader?: AuthHeaderField, accessJWT?:string): Promise<GraphQLResponse> {
   const headers: Record<string, string> = { "Content-Type": "application/json" };
+  // add graphql auth header is defined
   if (authHeader && authHeader.key && authHeader.value) {
     headers[authHeader.key] = authHeader.value;
   }
+  // add user access token (login to tenant) if defined
+  if (accessJWT && accessJWT!='') {
+    headers["X-Dgraph-AccessToken"] = accessJWT;
```

Comment:
do we require the if condition? We can just set the header otherwise also

Reply:
not sure of the behavior with a header set to ''.
I have not tested so I prefer to keep the test